### PR TITLE
WIP PR for BXMSDOC-8474: Release notes docs for RHPAM/RHDM 7.13

### PR DESCRIPTION
**Pull request by kaldesai**
**Epic:** https://issues.redhat.com/browse/BXMSDOC-8474

**Doc previews:** 
[Release notes for Red Hat Process Automation Manager 7.13](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-8474-RHPAM-RN-7.13/)
[Release notes for Red Hat Decision Manager 7.13](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-8474-RHDM-RN-7.13/)

### CODE REVIEW

File: doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc [hunk 1]
```diff
@@ -3,191 +3,54 @@
 
 This section highlights new features in {PRODUCT} {PRODUCT_VERSION}.
 
-ifdef::DM[]
+== Ability to only enable decision management components and functionality
 
-== {PRODUCT} {ENTERPRISE_VERSION} Kogito decision services support
+You can now configure the deployment to only enable decision management related components and functionality so that jBPM and case management features are disabled.
```
Comment (krisv) [inline doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc:8]: Shouldn't we be more specific?  This relates to the fact that we now only have one set of images/operator for both RHPAM and RHDM, so there a new property on the templates and operator so you can run those with only DM capabilities enabled.

File: doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc [hunk 2]
```diff
@@ -3,191 +3,54 @@
...
 
-=== IBM Power support in {OPENSHIFT} 4.10
+== Ability to use a custom registry and specific images
 
-Starting with {PRODUCT} {ENTERPRISE_VERSION}, IBM Power supports {OPENSHIFT} 4.10 using Open JDK 11. For a complete list of supported configurations, see https://access.redhat.com/articles/3354301[Red Hat Decision Manager 7 Supported Configurations].
+You can now configure the custom registry to use specific images during the configuration of the environment.
 
-endif::DM[]
+== Ability to output error logs
```
Comment (krisv) [inline doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc:14]: I don't think this title matches the content well, I would suggest something like: Eager initialization of Kafka emitters for jBPM

Comment (krisv) [reply doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc:14]: This should also not be in the RHDM release notes

File: doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc [hunk 3]
```diff
@@ -3,191 +3,54 @@
...
 
-== Ability to send cookies in the REST work item handler
+You can now set the `org.kie.jbpm.event.emitters.eagerInit` property to `true` to output the error log when {KIE_SERVER] starts and initializes the Kafka emitter.
 
-You can now send cookies to the REST work item handler using the `Cookie` header and cookie parameters in a REST API as shown in the following example:
+== Support for Quarkiverse File Vault and keystore files
```
Comment (krisv) [inline doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc:18]: Shouldn't the title mention this is in the context of PIM?

Comment (krisv) [reply doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc:18]: This should also not be in the RHDM release notes

File: doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc [hunk 4]
```diff
@@ -3,191 +3,54 @@
...
 
-`cookieParam1=cookieParam1_Value;cookieParam2=cookieParam2_Value`
+You can use the Quarkiverse File Vault extension to store credentials as keystore files and use the `file` method to use the keystore files with the Process Instance Migration (PIM) Keystore Vault.
 
-You can also define the path for which the cookie is valid using the `CookiePath` header. By default, the value of `CookiePath` is set to `/`.
+== Ability to use a secret for Admin user during KieApp configuration
```
Comment (krisv) [inline doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc:22]: There's a few items that seem to be related to configuring the cloud-based setup.  Would it make sense to bundle them under one title, like we do for DM designer?

File: doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc [hunk 5]
```diff
@@ -3,191 +3,54 @@
...
+== {PLANNER}
 
-When deploying {PRODUCT} on {OPENSHIFT}, running post-configuration scripts on {EAP} on the {CENTRAL} and {KIE_SERVER} pods is now supported. You can use these scripts to configure Elytron users.
+Additional metrics are available for monitoring the {PLANNER_SHORT} solver with Micrometer.
 
-=== Role mapping configuration for {OPENSHIFT} without a file
+== Spring Boot
 
-When deploying {PRODUCT} on {OPENSHIFT}, you can now configure role mapping for {RH-SSO} or LDAP configuration by entering the configuration directly in the settings, without using a role mapping configuration file. The use of a role mapping configuration file is now optional but still supported.
+The start.jpbm.org website is no longer available. To create Spring Boot business applications use Maven archetype commands.
```
Comment (krisv) [inline doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc:56]: This should also not be in the RHDM release notes

File: doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc (hunk 2 above)
Comment (kaldesai) [reply doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc:14]: Noted

File: doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc (hunk 3 above)
Comment (kaldesai) [reply doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc:18]: Mentioned PIM in the title

**Pull request merged**